helpdesk_mgmt/views/helpdesk_ticket_views.xml

```diff
@@ -145,9 +145,13 @@
                     </div>
                     <group name="main">
                         <group>
-                            <field name="active" invisible="1" />
+                            <field name="active" attrs="{'invisible': 1}" />
                             <field name="team_id" options='{"always_reload": True}' />
-                            <field name="user_ids" invisible="1" readonly="1" />
+                            <field
+                                name="user_ids"
+                                invisible="1"
```

Comment:
attrs="{'invisible': 1} ??

Reply:
We have to move to attrs only when the condition is dynamic. In this case, it's static and it's correct as is.

Reply:
Thanks for the info

Reply:
You're right, I forgot it, thanks!

Reply:
@dalonsod please revert the change, as is not needed and the syntax is incorrect: attts expect a domain, not a simple value

Reply:
@pedrobaeza done!